kubernetes/ansible/roles/logging/templates/filebeat.yaml

```diff
@@ -8,6 +8,25 @@ daemonset:
       - type: container
```

Comment:
Move this to the end of this list. This pattern will match every container log in the cluster and will not consider the below patterns. In case we run both flink and non flink pods in same cluster.

Reply:
moved the config to the end of the list